pipeline/run_models.py

```diff
@@ -40,14 +44,23 @@ def runner(model_jobs: list[Job], formatter: SKFormatter) -> dict[str, pd.Series
     date = datetime.today().strftime("%m_%d_%H_%M")
     path = f"/share-files/runs/{date}/{date}_"
 
-    x_train, x_test, y_train, y_test = formatter.generate_train_test_split()
+    # Obtain train and test data
+    x_train, _, y_train, _ = formatter.generate_train_test_split()
+    _, x_test, _, y_test = SKFormatter(
+        "/share-files/pickle_files_features_and_ground_truth/2013.pkl",
+        test_size=1.0,
+        discard_features=formatter.discard_features,
+    ).generate_train_test_split()
```

Comment:
Kan vi ikke give x_train, y_train, x_test, y_test som parametre, i stedet for at gøre det her i run? 
Derudover, så skal test_size sættes til 0, hvis du gerne vil have alt data til x_train og y_train.

Reply:
Hver opmærksom på, at vi har to formatters. 
Den ene giver x_train, y_train (fra 2012, men test_size=0.0), mens den anden (den du har kommenteret på her) giver x_test, y_test (fra 2013, med test_size = 1.0)

Kan god rykke begge formatters ud i main, og så parse som parametre til runner().